### ISSUE TITLE
Link crawl features

### ISSUE THREAD

**Issue by MrFadiAi** [joschan21/quill#19]
We would love to have also a field to import link and converting it to pdf and send it to plannetscale database

is that possible ?

**Comment by MattBCarr** [comment]
Yes it is possible – you could use a library like pdf-lib to convert whatever it is you want into a PDF file
